qt/widgets/common/src/SequentialFitDialog.cpp

```diff
@@ -373,7 +372,61 @@ void SequentialFitDialog::functionChanged() { populateParameters(); }
 
 void SequentialFitDialog::finishHandle(
     const Mantid::API::IAlgorithm * /*alg*/) {
+  getFitResults();
   emit needShowPlot(&ui, m_fitBrowser);
+  m_fitBrowser->sequentialFitFinished();
+}
+
+/// Set the parameters to the fit outcome.
+/// if more than one entry in the gui the parameters used are first entry with
+/// same name as fit property browser if one entry will try and use row that
+/// corresponds to ws index in fit browser, if not found defaults to first row.
+void SequentialFitDialog::getFitResults() {
+  auto wsName = m_outputName;
+  if (Mantid::API::AnalysisDataService::Instance().doesExist(wsName)) {
```

Comment:
Can we change this to
```
if (!Mantid::API::ADS::Instance().doesExist(wsName)){
    // Possibly emit a warning if this should never happen?
    return;
```

As this reduces the nesting of everything else below by one layer